src/%IPM/Repo/Definition.cls

```diff
@@ -3,7 +3,7 @@ Include (%syPrompt, %IPM.Common)
 Class %IPM.Repo.Definition Extends (%Persistent, %ZEN.DataModel.Adaptor, %IPM.CLI.Commands) [ Abstract ]
 {
 
-Parameter DEFAULTGLOBAL = "^%IPM.Repo.Definition";
+Parameter DEFAULTGLOBAL = "^IPM.Repo.Definition";
```

Comment:
why?

Reply:
Hi Dmitry, good question! This is to make sure Repo definition's storage global are namespace specific. ^% is instance-wide globals and we don't want the repo definition to exist in other databases, so change it accordingly!

Reply:
We still don't have an agreement about this. I'm sure that this should be system-wide